src/download.test.ts

```diff
@@ -353,7 +353,7 @@ describe("getFileContent", () => {
       mock.onGet(expectedUrl).reply(200, {}, headersWithProof);
 
       await expect(client.getFileContent(skylink)).rejects.toThrowError(
-        "File content response invalid despite a successful request. Please try again and report this issue to the devs if it persists. Error: Expected returned skylink to be the same as input data link"
+        "File content response invalid despite a successful request. Please try again and report this issue to the devs if it persists. Error: Expected returned skylink ('AQDwh1jnoZas9LaLHC_D4-2yP9XYDdZzNtz62H4Dww1jDA') to be the same as input data link ('XABvi7JtJbQSMAcDwnUnmp2FKDPjg8_tTTFP4BwMSxVdEg')"
```

Comment:
personal opinion: I generally like to hardcode stuff in tests since it makes me feel less vulnerable to mistakes but I agree that we could have used `skylink` here since we already use it as a variable in the test anyway

Reply:
Yes, this is what I mean - I am fine with hardcoding stuff but if it's in a var then I always use the var. Otherwise I might come by later, change the value of the var and not notice the error message and then the error message is going to be very confusing for however happens to encounter a failed test there.

Reply:
Like Karol, I prefer to hardcode strings in tests. I understand your point though - it would be a bit cleaner with the variable. I'm going to resolve the thread, because it's preventing merging, but please open an issue if you feel strongly about it!